src/wp-admin/includes/update.php

```diff
@@ -933,3 +938,76 @@ function wp_recovery_mode_nag() {
 	</div>
 	<?php
 }
+
+/**
+ * Checks whether auto-updates are enabled.
+ *
+ * @since 5.5.0
+ *
+ * @param string $type    The type of update being checked: 'theme' or 'plugin'.
+ * @return bool True if auto-updates are enabled for `$type`, false otherwise.
+ */
+function wp_is_auto_update_enabled_for_type( $type ) {
+	switch ( $type ) {
+		case 'plugin':
+			/**
+			 * Filters whether plugins manual auto-update is enabled.
+			 *
+			 * @since 5.5.0
+			 *
+			 * @param bool $enabled True if plugins auto-update is enabled, false otherwise.
+			 */
+			return apply_filters( 'wp_plugins_auto_update_enabled', true );
+		case 'theme':
+			/**
+			 * Filters whether plugins manual auto-update is enabled.
+			 *
+			 * @since 5.5.0
+			 *
+			 * @param bool True if themes auto-update is enabled, false otherwise.
+			 */
+			return apply_filters( 'wp_themes_auto_update_enabled', true );
+	}
+
+	return false;
```

Comment:
Following up on https://github.com/WordPress/wordpress-develop/commit/86941f9f0979e9b879c38fa7e85410b9bedf1a2f#r39289435

Would be good to add

```
	case 'translation':
		return true;
```

to the switch.

Reply:
Not sure this is a good idea.
- The feature plugin is for auto-updating plugins and themes, there's nothing in it about translations.
- There is no UI for setting specific translation files to be auto-updated by the users.
- There is no option `auto_update_translations` (that would be needed/checked if `wp_is_auto_update_enabled_for_type( 'translation' )` returns true.